ui/src/sources/components/InfluxTable.js

```diff
@@ -0,0 +1,96 @@
+import React, {PropTypes} from 'react'
+import {Link} from 'react-router'
+
+import Dropdown from 'shared/components/Dropdown'
+
+const kapacitorDropdown = (kapacitors) => {
+  if (!kapacitors || kapacitors.length === 0) {
+    return (
+      <span>--</span>
+    )
+  }
+  const kapacitorItems = kapacitors.map((k) => {
+    return {text: k.name}
+  })
+  return (
+    <Dropdown items={kapacitorItems} onChoose={() => {}} selected={kapacitorItems[0].text} />
+  )
+}
+
+const InfluxTable = ({
+  sources,
+  source,
+  handleDeleteSource,
+  location,
```

Comment:
it should now work with the update to v3 of `react-router`, which was merged into master last night. our previous v2.4 did not support location props via `withRouter`.